requestor/README.md

```diff
@@ -51,8 +51,99 @@ Default name of this subnet is `erigon`.
 
 ## Use requestor server
 
+### Endpoints
+
 All requests must contain a header `{'Authorization': 'Bearer [ETHEREUM_ADDRESS]'}`
 
     GET  /getInstances      - list of all instances created by the user (includes stopped instances)
     POST /createInstance    - create new instance, this requires "params" in body and also accepts optional "name"
     POST /stopInstance/<id> - stop instance with id <id>
+
+### Command-line interface
+
+You can use CLI to interact with the requestor server. First, generate a `keystore.json` file. You can export from yagna or use [vanity-eth](https://vanity-eth.tk/). Then you can list, create and stop erigon instances:
+```shell
+$ python3 -m venv .venv
+$ . .venv/bin/activate
+$ pip install -r requirements.txt
```

Comment:
I'd like it separate (make it clear it's "CLI initialization needed just once")
It'd be good idea to make it copy/paste-able to the terminal - without `$`

Reply:
Alright, I'll  put it in a separate block.